packages/@expo/prebuild-config/src/plugins/unversioned/expo-splash-screen/withIosSplashColors.ts

```diff
@@ -18,7 +18,10 @@ export const withIosSplashColors: ConfigPlugin<IOSSplashConfig> = (config, splas
   return withDangerousMod(config, [
     'ios',
```

Comment:
This should indicate the platform will only be ios

Reply:
I've just pushed a commit `fix(prebuild-config): make iOS splash-screen plugins no-op when called by macOS` to implement that 👍